3rdparty/carotene/src/pyramid.cpp

```diff
@@ -40,6 +40,7 @@
 #include "common.hpp"
 
 #include <vector>
+#include <cstdlib>
```

Comment:
This statement should be moved to "comon.hpp", because some other source files in Carotene need it too (e.g. _canny.cpp_).